device_operation/include/device_conv_fwd.hpp

```diff
@@ -32,6 +31,65 @@ struct DeviceConvFwd : public BaseOperator
                         OutElementwiseOperation out_element_op) = 0;
 
     virtual std::unique_ptr<BaseInvoker> MakeInvokerPointer() = 0;
+
+    static std::vector<ck::index_t>
+    ComputeOutputSpatialLengths(std::vector<ck::index_t> input_spatial_lengths,
```

Comment:
Please move this function out of this class, and implement it as a utility function for convolution

Reply:
moved it to device_operation/conv_utility/conv_utility.hpp